app/uk/gov/hmrc/tariffclassificationfrontend/views/templates/application_template.scala.html

```diff
@@ -116,154 +130,189 @@ <h2 id=@headerId>@messages(headerId)</h2>
+                        }else{
+                            @messages("pdf.application.section.applicationType.newApplication")
                         }
-                        </tr>
-                    </table>
+                        </td>
+                    </tr>
 
-                        @sectionHeading("pdf.application.section.aboutItem.heading")
-                    <table>
+                    @if(application.reissuedBTIReference.isDefined){
                         <tr>
-                            <td>@messages("pdf.application.section.aboutItem.description")</td>
-                            <td id="application.goods.details">
-                                @application.goodName <br/>
-                                <span class="pre-wrap">@application.goodDescription</span>
+                            <td>@messages("pdf.application.section.applicationType.reIssue")</td>
+                            <td id="application.reissuedBTIReference">
+                                @application.reissuedBTIReference <br/>
                             </td>
                         </tr>
-                        <tr>
-                            <td>@messages("pdf.application.section.aboutItem.confidentialInfo")</td>
-                            <td id="application.confidentialInformation">@if(application.confidentialInformation.isDefined) {
+                    }
+                    <tr>
+                        <td>@messages("pdf.application.section.aboutItem.description")</td>
+                        <td id="application.goods.details">
+                            @application.goodName <br/>
+                            <span class="pre-wrap">@application.goodDescription</span><br/>
+                            @if(application.confidentialInformation.isDefined) {
                                 <span class="pre-wrap">@application.confidentialInformation.get</span>
-                            } else {
-                                @messages("site.no")
-                            }</td>
-                        </tr>
+                            }
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>@messages("pdf.application.section.aboutItem.uploadeDocuments")</td>
+                        <td id="application.attachments">@if(attachments.isEmpty) {@messages("site.none")} else {
+                            @for(row <- attachments) {
+                                @row.fileName <br/>
+                            }
+                        }
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>@messages("pdf.application.section.aboutItem.importOrExport")</td>
+                        <td id="application.importOrExport">
+                        @{application.importOrExport match {
+                            case Some("IMPORT") => messages("pdf.application.section.aboutItem.import")
```

Comment:
Would this ever need to be a case insensitive match?

Reply:
It's an Enum I think

Reply:
Should the match not be on the enum value anyway?

Reply:
I say 'yes' to Ed's last comment

Reply:
done